lxd/metrics/types.go

```diff
@@ -16,10 +16,10 @@ type MetricSet struct {
 type MetricType int
 
 const (
-	// CPUSecondsTotal represents the total CPU seconds used.
-	CPUSecondsTotal MetricType = iota
```

Comment:
By re-ordering these you have changed all of their values. Is this intentional? Are there any side effects?

Reply:
I just reordered to make it more organized, no other purpose. As for side effects, I checked that they are always being referenced by their name and are never exposed to the user so the underlying value changing should not have any side effects.